evolution-chain/precompile/laos-evolution/contracts/LaosEvolution.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.3;
+
+/// @title Pallet Collection Manager Interface
+/// @author LAOS Team
+/// @notice This interface allows Solidity contracts to interact with pallet-living-assets
+/// @custom:address 0x0000000000000000000000000000000000000403
+interface LivingAssets {
+    /// @notice Event emitted when a new collection is created
+    /// @param collectionId Id of the newly created collection
```

Comment:
very minor, it reads a bit weird, here and in line 16. Typically you'd introduce a "the", that would go like: 

/// @param collectionId the id of the newly created collection
/// @param owner the address that owns the newly created collection